lib/rules/require-default-props.js

```diff
@@ -105,6 +106,13 @@ module.exports = {
       if (node.type === 'Identifier') {
         return findVariableByName(node.name);
       }
+      if (
+        node.type === 'CallExpression' &&
+        propWrapperFunctions.has(node.callee.name) &&
+        node.arguments && node.arguments[0]
```

Comment:
in v8, accessing an `arguments` index that's greater than `length` deoptimizes, so it'd be a performance issue.